redisinsight/api/src/modules/database/repositories/local.database.repository.ts

```diff
@@ -90,6 +92,17 @@ export class LocalDatabaseRepository extends DatabaseRepository {
    * @param database
    */
   public async create(database: Database): Promise<Database> {
+    // Do not create a connection if it triggered from cloud and have the same fields
+    if (database.cloudDetails) {
+      const existingEntity = await this.findEntity(
+        database,
+        ['host', 'port', 'username', 'password', 'tls'],
```

Comment:
"tls" here just a flag. Not sure that it is enough... certs & key body should be checked too, no?

Reply:
added certs and key body